plugins/woocommerce/src/Blocks/Patterns/PTKPatternsStore.php

```diff
@@ -179,23 +181,41 @@ public function fetch_patterns() {
 
 		$this->flush_cached_patterns();
 
-		$patterns = $this->ptk_client->fetch_patterns(
+		$dotcom_patterns = $this->ptk_client->fetch_patterns(
 			array(
 				'categories' => array( 'intro', 'about', 'services', 'testimonials' ),
 			)
 		);
+		if ( is_wp_error( $dotcom_patterns ) ) {
+			wc_get_logger()->warning(
+				sprintf(
+				// translators: %s is a generated error message.
+					__( 'Failed to get Dotcom patterns from the PTK: "%s"', 'woocommerce' ),
+					$dotcom_patterns->get_error_message()
+				),
+			);
+			return;
+		}
 
-		if ( is_wp_error( $patterns ) ) {
+		$woo_patterns = $this->ptk_client->fetch_patterns(
+			array(
+				'site'       => 'wooblockpatterns.wpcomstaging.com',
```

Comment:
In the future, will we replace this URL with the production site? 

Could it make sense create a quick issue not to forget to do it?

Reply:
As far as I know, the decision regarding where the patterns are going to be is still under discussion, so for now, we are putting them in `wooblockpatterns.wpcomstaging.com` to make progress. I'll create an issue to make sure we don't forget to review the decision.

Reply:
> the decision regarding where the patterns are going to be is still under discussion, so for now, we are putting them in wooblockpatterns.wpcomstaging.com to make progress

Yes: The main source site needs to make an important change to support WooCommerce patterns.